src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -134,7 +134,15 @@ public boolean hasLog10PError() {
      * @return the -1 * log10-based error estimate
      */
     public double getLog10PError() { return log10PError; }
-    public double getPhredScaledQual() { return getLog10PError() * -10; }
+
+    /**
+     * Floating-point arithmetic allows signed zeros, +0.0 and -0.0. Adding the constant 0.0 to the
+     * result ensures that the returned value is never -0.0 since (-0.0) + 0.0 = 0.0.
+     * See issue https://github.com/broadinstitute/gsa-unstable/issues/841 for details.
```

Comment:
```htsjdk``` is a public repository while ```gsa-unstable``` is not, so this reference is not helpful.  Can you provide the details here?